src/data/services/EcommerceApiService.js

```diff
@@ -18,7 +18,9 @@ class EcommerceApiService {
       ...options,
     });
     const url = `${EcommerceApiService.ecommerceBaseUrl}/api/v2/enterprise/coupons/${enterpriseId}/overview/?${queryParams.toString()}`;
-    return EcommerceApiService.apiClient().get(url);
+    return EcommerceApiService.apiClient({
+      useCache: true,
```

Comment:
Should this use `configuration.USE_API_CACHE,` as is shown below in `fetchCoupon`?

Reply:
oops this one is not meant to be cached.